SwanCustomEnvironments/swancustomenvironments/scripts/builders/conda.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# Create environment and install packages
+${BUILDER_VERSION} env create -f ${REQ_PATH} -n ${ENV_NAME} -y
```

Comment:
Does it make sense to create here empty and install packages later? To keep the same structure as the others? Or not really worth it?

Reply:
Maybe not, never mind.